db/config.c

```diff
@@ -368,7 +368,7 @@ static char *legacy_options[] = {
     "init_with_queue_ondisk_header off",
     "init_with_queue_compr off",
     "usenames",
-    "json_escape_control_characters off",
+    "json_escape_control_characters on",
```

Comment:
This tunable is enabled by default anyway (https://github.com/bloomberg/comdb2/blob/7.0/lua/sp.c#L93), may be just remove it from this list altogether.

Reply:
Yeah good catch.